_sass/blocks/_landing.scss

```diff
@@ -0,0 +1,85 @@
+.landing-section_top,
```

Comment:
Selector `landing-section_top` should be written in lowercase with hyphens<br>Selector `landing-section_open` should be written in lowercase with hyphens